VideoLocker/res/values/text_styles.xml

```diff
@@ -103,19 +103,27 @@
     <style name="bold_white_text" parent="@style/bold_text">
         <item name="android:textColor">@android:color/white</item>
     </style>
+
+    <style name="regular_black_text" parent="@style/regular_text">

```

Comment:
Instead of "black" maybe it should just be "dark content" or something? We don't typically use 0x000000 black, instead using the darkest edx color.

It could also be something more semantic like "body text"?